### ISSUE TITLE
There are issues with the handling of querying types

### ISSUE THREAD

**Issue by fxzxmic** [natesales/q#83]
```
Dev[~]$ q Gateway.NAT.local
Gateway.NAT.local. 0s A T.local.        0       IN              192.168.138.1
Gateway.NAT.local. 0s AAAA fc00::c0a8:8a01
```

**Comment by fxzxmic** [comment]
```
Dev[~]$ ~/Downloads/q nat.local -R
nat.local. 1m A 127.0.0.1 (NAT.local.)
nat.local. 1m AAAA ::1 (NAT.local.)
nat.local. 1m AAAA ::ffff:127.0.0.1 (NAT.local.)
Dev[~]$ ~/Downloads/q NAt.local -R
NAt.local. 1m AAAA ::1 (NAT.local.)
NAt.local. 1m AAAA ::ffff:127.0.0.1 (NAT.local.)
FATA[0000] error reversing PTR record: dns: unrecognized address: t.local.      60      IN              127.0.0.1
```
What keyword does uppercase NA seem to be? When encountering uppercase NA, it is directly deleted, and no issues were found in the raw format of the DNS response.

**Comment by fxzxmic** [comment]
```
Dev[~]$ q NS.network
NS.network. 2h A 3.64.163.50
NS.network. 24h NS .network.    86400   IN              ns1.dan.com.
NS.network. 24h NS .network.    86400   IN              ns2.dan.com.
```
There is indeed a problem.

**Comment by natesales** [comment]
This is a fun edge case where the record type is a substring of the record label. Fixed in v0.19.2

```
$ q --version
https://github.com/natesales/q version 0.19.2 (0ca9b8b13fc69121144d85365c16b3e21508ffc3 2024-01-21T21:16:52Z)
$ q NS.network
NS.network. 1h51m13s A 3.64.163.50
NS.network. 11h45m43s NS ns1.dan.com.
NS.network. 11h45m43s NS ns2.dan.com.
```

Can you check nat.local again?

**Comment by fxzxmic** [comment]
Fixed, thanks.
